assets/src/domain/eventEditor/ui/tickets/ticketPriceCalculator/buttons/PriceModifierActions/styles.scss

```diff
@@ -0,0 +1,12 @@
+$btn: '.esprs-button.esprs-btn-default';
+$btn-size: 30px;
+
+.ee-price-modifier-actions {
+	#{$btn} {
+		margin-bottom: 0;
+		min-height: $btn-size;
+		min-width: $btn-size;
+		height: $btn-size;
+		width: $btn-size;
```

Comment:
@tn3rb we either need to increase input height or to make small buttons, to make it consistent:

![image](https://user-images.githubusercontent.com/1477453/78177016-18d01e00-7466-11ea-99b3-969c16fdcbca.png)

Reply:
they can both use `var(--ee-icon-button-size)`

Reply:
purfect, thx:

![image](https://user-images.githubusercontent.com/1477453/78241677-75285180-74e9-11ea-8f0f-98206bd859df.png)